src/Magnum/Math/Test/FunctionsTest.cpp

```diff
@@ -293,6 +297,15 @@ void FunctionsTest::ceil() {
     CORRADE_COMPARE(Math::ceil(2.7_degf), 3.0_degf);
 }
 
+void FunctionsTest::binomialCoefficient() {
+    CORRADE_COMPARE(Math::binomialCoefficient(1, 1), 1);
+    CORRADE_COMPARE(Math::binomialCoefficient(1, 0), 1);
+    CORRADE_COMPARE(Math::binomialCoefficient(19, 11), 75582);
+    CORRADE_COMPARE(Math::binomialCoefficient(1000, 999), 1000);
+    CORRADE_COMPARE(Math::binomialCoefficient(0, 0), 1);
+    CORRADE_COMPARE(Math::binomialCoefficient(32, 11), 129024480);
+}
+
```

Comment:
Thanks for the test!

I'm usually also testing assertion messages, to be sure that they make sense. An [example how to do that is here](https://github.com/mosra/magnum/blob/80a844c6c5256b1c77a04db22f8042891b521c0b/src/Magnum/Math/Test/VectorTest.cpp#L556-L568) -- note it has to be a separate test due to the skip (`binomialCoefficientOverflow()` e.g.).